components/channel_header_dropdown/mobile_channel_header_dropdown.tsx

```diff
@@ -14,18 +16,18 @@ import {ChannelHeaderDropdownItems} from 'components/channel_header_dropdown';
 import Menu from 'components/widgets/menu/menu';
 import MenuWrapper from 'components/widgets/menu/menu_wrapper';
 
-import MobileChannelHeaderDropdownAnimation from './mobile_channel_header_dropdown_animation.jsx';
+import MobileChannelHeaderDropdownAnimation from './mobile_channel_header_dropdown_animation';
 
-export default class MobileChannelHeaderDropdown extends React.PureComponent {
-    static propTypes = {
-        user: PropTypes.object.isRequired,
-        channel: PropTypes.object.isRequired,
-        teammateId: PropTypes.string,
-        teammateIsBot: PropTypes.bool,
-        teammateStatus: PropTypes.string,
-        displayName: PropTypes.string.isRequired,
-    }
+type Props = {
+    user: UserProfile;
+    channel: Channel;
+    teammateId: string | null;
```

Comment:
Shouldn't these be optional
  `teammateId: string | null;
    teammateIsBot: boolean;
    `

Reply:
`teammateId` should be optional since `getTeammateId` returns string or null.
https://github.com/mattermost/mattermost-webapp/blob/0fed29d3be851183ee613f7295e672e7fc33c51f/components/channel_header_dropdown/index.ts#L43

Reply:
@sk409 that’s what I meant `teammateIsBot: boolean;` should be optional as per previous code.

Reply:
I made `teammateIsBot` optional